Makefile

```diff
@@ -299,20 +302,25 @@ clobber: $(SPECIAL_CLOBBER)
 
 flash:
 	@echo "use one of the following targets to flash the firmware"
-	@echo "  make flash512k - for ESP with 512kB flash size"
-	@echo "  make flash4m   - for ESP with   4MB flash size"
+	@echo "  make flash512k     - for ESP with 512kB flash size"
+	@echo "  make flash1m-dout  - for ESP with   1MB flash size and flash mode = dout (Sonoff, ESP8265)"
```

Comment:
> ESP8265

What kind of SoC is that? Did you mean to write ESP8285 here?

Reply:
@jpeletier we can merge the PR once this comment has been addressed.

Reply:
@marcelstoer Yes, it was a typo. Fixed.